spec_mode/nirspec_fs.ipynb

```diff
@@ -122,7 +122,9 @@
    "cell_type": "code",
```

Comment:
<p>Since this notebook is written to work with version 1.1.0 of <code>jwst</code>, it could be worthwhile to raise a warning if a user is running a different version. To do so, you could add <code>import warnings</code> to your first import cell and then create a new cell with something like this:</p><pre class="ql-syntax" spellcheck="false">if jwst.__version__ != '1.1.0':
&nbsp;&nbsp;&nbsp; warnings.warn(f"You are running version {jwst.__version__} of the jwst "
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; "module instead of the intended 1.1.0.")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>